### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by bhastings1019**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,134 @@
...
-3. Create a parser instance, (see https://serialport.io/docs/api-parsers-overview)
-4. Assign the the BrightSign binding to the SerialPort
-5. For reading, pipe the SerialPort to the parser
-6. For writing, use the "write" method of the SerialPort. See the example in index.js, where the string "abc" is written
+This reference currently assumes USB-A is sending to USB-C serial protocol. It may be the case the Player's log output is over the 3.5mm Serial Port or SSH/Telnet. 
 
-This example includes a custom BrightScript (autorun.brs) that creates the Node.js application, but you can also create an HTML 5 widget in BrightAuthor. Remember to check the box "Enable node.js".
+## Getting Started
+---
+After cloning the repo and changing to it's directory, do the following.
```
Comment (blshukla) [inline README.md]:
```suggestion
After cloning the repo and changing to its directory, do the following.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,134 @@
...
 
+```bash
+npm i
+npm run build:dev
+```
+
+Deploy the bundled application to the player through the DWS APIs. See 
+[Enabling development tools on the Player ](#enable-player-development-tools) to enable the DWS, and [Deploying Code Remotely](#deploy-code) to deploy the code.
```
Comment (blshukla) [inline README.md]: The links on this line do not work.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,134 @@
...
+```bash
+ls /dev/tty*
+```
+
+For example, the USB port might be /dev/tty.usbserial in which case you can execute a command like,
+```
+screen /dev/tty.usbserial 115200 
+```
+
+Once connected, use the serial software of your choice (on linux and Mac, the "screen" command is quite popular and all you need).
```
Comment (blshukla) [inline README.md]: We should also add telnet/ssh instructions for Windows here.

File: README.md [hunk 4]
```diff
@@ -1,21 +1,134 @@
...
+export PLAYER_PW={the pw for that player, by default it's serial number}
+```
+
+The file scripts/put uses the DWS API to move files to the player.  If your application builds additional files you will need to add them to that script in the line that lists the files to copy.  For example:
+
+```bash
+for f in dist/*.html dist/*.js src/autorun.brs
+```
+
+The current script copies all the html and js files in the dist folder, and the autorun from the source file.  If your app needs more files, add them to that line.
```
Comment (blshukla) [inline README.md]: This and the previous line are almost similar. Can we merge them?

File: README.md [hunk 5]
```diff
@@ -1,21 +1,134 @@
...
+```
+
+You should see the results of the webpack build, followed by a list of the files copied to the player.
+
+### Starting the App on the Player
+
+Just copying the files will not restart your application on the player.  From a serial or Telnet / SSH session to the player:
+
+Restarting the application can be done via, 
+* rebooting the player (`npm run rp`)
```
Comment (blshukla) [inline README.md]: We can also mention Reboot player using Control tab in DWS, if applicable.

**Review by blshukla — COMMENTED** [review]
I have a few comments about the README notes. I assume the code has been well tested and runs as per the notes.

File: README.md (hunk 2 above)
Comment (bhastings1019) [reply README.md]: I think github doesnt render markdown anchor links well in the raw view - could you switch your files edit view to the "Display rich diff" view (the button to right of the <> button at the top of your window). Do the links render better?

File: README.md (hunk 1 above)
Comment (bhastings1019) [reply README.md]: Should be updated.

File: README.md (hunk 3 above)
Comment (bhastings1019) [reply README.md]: Thanks - I added a minor note. There's work being done to incorporate a more complete set of instructions to be published online.

File: README.md (hunk 4 above)
Comment (bhastings1019) [reply README.md]: Good shout - merged.

File: README.md (hunk 5 above)
Comment (bhastings1019) [reply README.md]: Yes - good looks, updated this bit.

**Comment by bhastings1019** [comment]
The branch has been updated to point to the originally intended commit; previous branch was out of date.

**Review by lherlein — APPROVED** [review]

**Pull request merged**